external/gsOpenNurbs.cmake

```diff
@@ -0,0 +1,66 @@
+## Version: $Id:  $
+##
+##
+
+## Commentary:
+##
+##
+
+## Changelog:
+##
+##
+
+##
+## Code starts here
+## #############################################################################
+
+find_package(opennurbs QUIET)
+
+if(NOT opennurbs_FOUND)
+
+    message(STATUS "Opennurbs not found, fetching ...")
+
+    include(FetchContent)
+    include(GNUInstallDirs)
```

Comment:
With the additions of FetchContent/GNUInstallDirs the minimum version of Cmake required should be revised to a higher number